src/database/courses/index.tsx

```diff
@@ -65,6 +65,94 @@ const courses = [
pAldwZVV1Wa4rSVKUZl1vPpkYe2NCLSFQUE/VofUWMBtJpvxhM+aOpetQbFEUuGlqHN8NgriD6iJDIVVeSdWnA9Z9QQCjGs9+8ffvrXG3ByAm1Krx83YTI4dH+4eE+cRiRILCobgMJqLpkic2qqtKlQEJYuJX5ebW77FbkOWjzUC7F1VVVtak2XW6X0u3WFGIWaU3oF8jIA7PnzZio6KskV6CKJuo+MS0I1TC0+PwpX1Twp/z1tECpyAPrUH5BTV+oO30PXKjNv/7s7dtnv83VjKb1ah7YkK4fqgNB3FklDjoNFFhqt9Gl1Kzk4ohMo9noQlw23FKXa7pcqqRKD1ySAhei1pW6jWbT3c1AcMPnM02qpjaaVLfGmvFAvYrwYWFdgESnN5UheLl1re8BDWOoLwJ3nnZRSxm9LVC1EYqd6RB2uE6++exh03hK7pUq3EtEf9HrGGQ0qCS8lr5ZWId6gn5iUcLTKwyBSgtPcSzL6+vweRZtgja5qRx4acaV7plBb429+e2xBERFhDhymSjDNw4m9FZiqIaoLOiFeDzMVWFLGakZY2pcYWgVjhCRbw/1Soy8i3c3s2rzAeTC7NjNQ6McKC1PuetznZEa0qqZkS0S1VW9F99L5GANqvDqwepa5/Ad6ktSikzcl5rq3Mx8d1xHMRSwMUo0jsUoA6omC7ZWha9JElkjOvuHnYOD1cND8D4eqqnSzCgNpVlzjdJs6MMohLN3JYTppnmZ9XbgdcuPDlFTev9oP9OJUCjM3W5CzbhqWXX+/rgxPM6A6pR90pSA9Pu5hulx6aFVOIJKSESvI1rt5bX+HE8Z9d2GY3JIn+15pWrTVdCMgOSDud/mJbNf67pShWenau4VDIeztEBgFzPEHrt3hga32GmPCJnyQHJhrmZ6VLBfQF6en6q9/RgIyC5rVZhdXDw7e3x2trgI1/P+KK1kE5jLgeRgQ4cKyDcvciClNi1sSRgIKDW0kXHWs3i2uLcI/9hb8sCRQM2Y5Q9FZHbqw/i9GIUwq5vBAtDb2duDdf5jEJCQPgjIK1YmwXHuC/fox6OhQI7XOnYcGk7VtxIDFkOx6Mb65WFvtYjwV04/0tF7QkcR1JnmK96N+ko9vE54CS4ZEhJegecEzguCagsOjTSEhJsbCZ/2zISrw5Krbzr7BweHq28O3GCl4EuI9XRdrKYFQvhnKrEbFFPpaGo3JAjr0dBuoh71+oJiXbjxZux1AUnaPIPM1wQct2sxIkZduWujamuRw8PDTmftW3QHgQ8lokl4hMQgzyX9YTGREkVxRVxJJhLRsG8lWY3yG7tR4cZ2xDUBSXXOPK4B9utFxB4FjfvC/R4YOYiAB66uHugeKIopkQDZkkFe8Nf5dc6XCtSFKHhgQAzX15PIAwN1NDA5nAECmo9AaoiAhJTNjiHH6BhO7bjqgdARiqD+kJYDCUFAo6roRhOBxgjRBBBOnwXCoTWUAyEp3txTt0RAflBf2ArGmlwU6RmU5MJ9c2Z66JsAcwPWeODV6mcVowqojUnuH3wYk+QC1bA24aWuTYHh0PQZaNWgkf2QT9A2GZ2KNQLaxYghHDlY67xbi+wfvdtfe4dmF0DpqPijvmqlKuz6UtVUVfSJ4TRfTaf9G+KKzx/wpV0GY7KDigjPmoJghxURgs/Y44JGXbkrzRioIEjAdx0oxd+iITXwwGQUCu+GpqMvkFyp+wQv5w+I0Yq/AgJWhKZBMrrejFlQpceLZthrqoPmevR35azFSMDs1SrciRwdRVYzHeLofP5OPZgmvLw+H6YS9nKJBHRPKl4ixIVDG8FwmDIKyAENadq9+CczPB58iwkEZBt3qwoP1kCfAQgpT58kczFdhvgwWebKLJjBDBHwnjEg4OC/1HjnqrChF02CKQE9iBEEvGNVmLJyQMOEgJ7jXNzppDc95gW0C6OeyLKWSaiuhS5oKKBnM+6ktzY3t5xbHtMCZu5GX5ggtEIxK9+iBx7HnQvHKH5BSPMCSo0RpudMgOEkc31h5d1hXcClYQIeO+PHnnNPpM0LyPaazD+yzHrDSeZ9Q6SWoAk4868ngwU8djouVPM4N83nQKnHAXnp14dWWX9Xvqk08/a7HgUvBfTEL8LWcy/uGKWI9F7zzLNvTiyy/u58U6lXwUsBj7XCsQUrOWcO5UGPWQF74XnLFLRtasejh8M4yT1BLH1xqWBPCGsCHuccuS1ozBx7Fnq98FLApSe9LP375OHDvgOenHxmkYKGzZgpzc569PlnQ/ninGcvn1wT0LPl1BbPjzc9nnjO09eS+SDg0tMv+tD2euUob3+3JA8a94WnU3/53/7z+TC+1Hj68tnTpWsCIgWhCHu2ZpZQOOe2HPc2N+95rnrgqy/7+O/nn//xxx99B/n97VeWlGLbQph/NIy5hSXgydNnTweFMASuc/N4xvk8vrVwHM89d+Y2txxb13LgUi+k6+EjKZvpOcbJV//7yppCfFeKyFKvfv0NaVBwYSsHMi48dx7nchDDntyIVfiRZfrdGQFf9erX74E5B5INivC9HLlFk2jb5mhVmH9tmX53Z470q96uSG8OnMk9P17YioOAm1BE9PGEY2MB+0akI68t64rYVURu4Oa+8KbzuSe+ec+hiWa+Ic0vZ3v7wtYFkmEz5rYF3HJuoR7ItbFAIw/MZq2fHo8wrMLWjwoZjMbkHFeHAU15YLZxJ2Ym3NIfnegrIsPk+zhGpG1Q0JJ7IpZbfYGBgNkBX7eYNhYJaNON4ZsFzDSkq7OLps9EtzWHCZhdXr7+hWEruFlA9tYENHljfbgHQvPBljJs1A5krS/DAwQka25TZAdbrzdj7CnDd2VE+qoEE8xPvajCd2JujB182nOkbQALOCFYwAnBAk4IFnBCPlEBKduw4g9xz9pm/fywpqiNTP1/5yZVG61Xh9hgI9PW72O3HoPBYDAYDAaDwWAwGAwGg8FgMBgMBoPBYDCYT5L/A/H5y6fjJbT6AAAAAElFTkSuQmCC",
```

Comment:
Could you please replace the image with a more suitable one sourced from a reliable and secure platform for comparison? It's important to prioritize security and trustworthiness when selecting images. Additionally, to avoid potential debugging and maintenance issues, let's refrain from using excessively long image URLs. We can consider downloading the image and adding it in a manner that doesn't complicate the code or increase character count unnecessarily